spec/e2e/enum_spec.rb

```diff
@@ -70,6 +70,16 @@
       expect(ids).to include(file2.id)
     end
 
+    it 'supports arrays' do
```

Comment:
Why did you define a new test instead of using the existing (skipped) one? Is it intentional?